### ISSUE TITLE
Cannot use keyboard shortcut after moving layer (Firefox)

### ISSUE THREAD

**Issue by juliandescottes** [juliandescottes/piskel#346]
**Firefox only !**

Steps to reproduce : 
- open Piskel in Firefox
- create a sprite (pen tool should be selected by default)
- click on "create layer"
- click on "move layer down"
- press "S"

*Expected* : the Rectangle selection tool should be selected
*Actual* : the Pen tool is still selected

After clicking anywhere else on the page, the keyboard shortcut will work again.

**Comment by captainbrosset** [comment]
I investigated this a little bit.
When the key is pressed, the `ShortcutService`'s keydown handler isn't even called.

I thought this was because the button was focused after being clicked and was consuming the event. But if you try with a slider for example (an input range, there are a couple in the right hand-side drawer), then even when it's focused, the event still bubbles up to the window.

I found out that if you create more than 2 layers, then it works fine after clicking the move layer down button just once. So I think this has something to do with the fact that the button is disabled:

after you press "move layer down", the new layer you created becomes the bottom-most one, and so the button gets disabled. And it seems that on Firefox, when a disabled button has focus, then all keydown events are consumed and lost.

So one way to fix this is blurring the button right before disabling it.
